fake-snippets-api/lib/public-mapping/public-map-package.ts

```diff
@@ -31,7 +32,7 @@ export const publicMapPackage = (internalPackage: {
     latest_package_release_id:
       internalPackage.latest_package_release_id ?? null,
     latest_version: internalPackage.latest_version ?? null,
-    license: internalPackage.latest_license ?? null,
```

Comment:
This is just an alias on selecting, similar to `latest_version` or `latest_package_release_id`. I don't think this will cause any problem and it's best if we follow how the registry is doing it maybe.

<img width="740" alt="Screenshot 2025-04-11 at 15 12 38" src="https://github.com/user-attachments/assets/8c6da8f5-64ee-41ec-8046-d3a639621926" />